_data/events.yml

```
@@ -224,19 +224,78 @@
     title: View meetup event
     target: _blank
 
-- title: Coding Club with Women Coding Community
+- title: |
+    Coding Club with Women Coding Community
   description: |
-    Learn Python 101: In subsequent sessions, we will delve into solving basic algorithmic problems, using "Grokking Algorithms" as our reference book, and thoroughly cover the fundamentals of the language.
-  category_style: coding-club
-  category_name: Coding Club
-  date: WED, SEP 05, 2024
+    We are excited to announce our next Coding Club session as a part of Women Coding Community!
+  category: Online Event
+  category_name: Online Event
+  date: Thu, Sep 5, 2024
```

Nitpick, but we've mostly added this in uppercase letters. 
```suggestion
  date: THU, SEP 5, 2024
```
I think these new additions were from running the python script, so maybe we would just add a CSS style to capitalise the display instead🤔

---

good point that would be a good idea.